src/classes/UTIL_UnitTestData_TEST.cls

```diff
@@ -84,6 +86,34 @@ public class UTIL_UnitTestData_TEST {
         
         return openStage;
     }
+
+    public static String getClosedTaskStatus(){
```

Comment:
Generally we add to this file when the methods are broadly applicable across the test suite, I think these Task related methods will be used only for the Engagement Plans feature, so I might move them to my EP_Utils class once it's merged in.